### PR TITLE
changelog for v4.12.0..master

### PR DESCRIPTION
**Pull request by andynu**
@bf4 on #292 you were asking for help with the changelog. Here's a summary of v4.12.0..master. There is a link to a comparison in the changelog that references v4.13.0 which is what I presume is next, but perhaps you consider these changes smaller. I did not change the VERSION.

### CODE REVIEW

**Comment by bergholdt** [comment]
@andynu we need to lock version of addressable and rainbow for 1.9.2 and 1.9.3 support.

I send you a pull request https://github.com/andynu/metric_fu/pull/1

File: HISTORY.md [hunk 1]
```diff
@@ -10,7 +10,24 @@ As such, a _Feature_ would map to either major (breaking change) or minor. A _bu
 * Features
 * Fixes
 * Misc
+
+### [4.13.0](https://github.com/metricfu/metric_fu/compare/v4.12.0...v4.13.0)
```
Comment (bf4) [inline HISTORY.md]: ✂️ only add this when releasing :)

Comment (andynu) [inline HISTORY.md]: Pardon, the link or the entire title?

Comment (bf4) [inline HISTORY.md]: @andynu The entire line.  Maintainers will add the version when releasing. Until release, it's correct o keep under 'master' which could be renamed to 'unreleased'

Comment (andynu) [inline HISTORY.md]: Very good. Thank you for the clarification.

**Pull request merged**